client/modules/IDE/components/ShareModal.jsx

```diff
@@ -1,37 +1,110 @@
 import React, { PropTypes } from 'react';
+// import AssetList from '../components/AssetList';
+function copyToClipboard() {
```

Comment:
This works, but doesn't take advantage of everything that React gives you. You can store the links as part of the ShareModal class, i.e. `this.embedLink = '...'`. Then, copyToClipboard should be a method of ShareModal rather than outside of the class. 

You can use a `setTimeout` and React state/setState to make the tooltips disappear.